Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -0,0 +1,41 @@
+Enable Hubble
+==============
+
+Hubble is a fully distributed networking and security observability platform
+for cloud native workloads. It is built on top of Cilium and eBPF to enable
+deep visibility into the communication and behavior of services as well as the
+networking infrastructure in a completely transparent manner.
+
+Deploy Hubble using Helm:
+
+.. parsed-literal::
+
+   helm upgrade cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.hubble.enabled=true \\
+      --set global.hubble.listenAddress=":4244" \\
+      --set global.hubble.metricsServer=":54321" \\
+      --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \\
+      --set global.hubble.relay.enabled=true \\
+      --set global.hubble.ui.enabled=true
+
```

Comment:
we should add a warning here that this is opening up a port 4244 for hubble-relay, might need @Rolinh 's help to properly phrase the warning.

Reply:
unix domain socket only by default.

with hubble-relay we need to specify `--set global.hubble.listenAddress`. the address needs to be reachable by hubble-relay, we typically do `":4244"`.

Reply:
And what about updating the guide to do this in two (or three) steps?
What I'm thinking of is:
step 1: enable hubble
step 2: enable hubble on port 4244 + deploy hubble relay
step 2 (or 3?): deploy hubble UI

This would allow us to clearly state what you get at each step and add appropriate warnings for step 2+ so that users may stop at step 1 and still get the benefits of local hubble.